### PR TITLE
commiting the changes in resource page

### PR DESCRIPTION
**Pull request by vinrusty**
## Issue that this pull request solves

 Closes: #760

## Proposed changes
1. Added cards in the resource page of the admin dashboard which contain data of the users by making GET request to the given API.
2.  Added delete button of suitable UI to each card.

### Brief description of what is fixed or changed
1. Made GET request to the route "/resources/getresources" using the *axios* library.
2. The fetched data is rendered in the form of cards each containing the data of that particular resource.
3. Added delete button and a button to the URL of the given resource.

## Types of changes

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] My changes does not break the current system and it passes all the current test cases.

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface
Screenshot (86)
## Other information

Any other information that is important to this pull request

### CODE REVIEW

File: frontend/src/pages/Admin/Components/Resource/resource.module.scss [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+.expire-update-cards {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  text-align: center;
+  color: rgb(41, 41, 41);
+}
+.expire-update-card {
+  background: linear-gradient(45deg, rgb(144, 144, 236), rgb(165, 178, 255));
```
Comment (Kajol-Kumari) [inline frontend/src/pages/Admin/Components/Resource/resource.module.scss:81]: Please make the background-color same as the broadcasts page in order to maintain the consistency.

File: frontend/src/pages/Admin/Components/Resource/resource.module.scss [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+.resource-card:hover {
+  box-shadow: 0px 2px 2px 2px gray;
+}
+
+.resource-card > h2 {
+  text-align: center;
+  color: #fff;
+}
+.resource-card > h4 {
+  text-align: center;
```
Comment (Kajol-Kumari) [inline frontend/src/pages/Admin/Components/Resource/resource.module.scss:22]: please make the whole content of the card centered aligned.

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@vinrusty The UI looks amazing. Please make the minor requested changes and it will be good to go :)

File: frontend/src/pages/Admin/Components/Resource/resource.module.scss [hunk 3]
```diff
@@ -0,0 +1,111 @@
+.resource-card-wrapper {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(350px, 1fr));
+  gap: 10px;
+  padding: 1.5rem;
+}
+
+.resource-card {
+  padding: 2rem;
+  // background: linear-gradient(45deg, rgb(113, 113, 245), rgb(71, 71, 219));
```
Comment (Kajol-Kumari) [inline frontend/src/pages/Admin/Components/Resource/resource.module.scss:10]: Please remove the commented line

File: frontend/src/pages/Admin/Components/Resource/Resource.jsx [hunk 4]
```diff
@@ -1,9 +1,69 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
+import { END_POINT } from "../../../../config/api";
+import axios from "axios";
+import "./resource.module.scss";
+import style from "./resource.module.scss";
 
 export function Resource() {
+  const [resourses, setResources] = useState([]);
```
Comment (Kajol-Kumari) [inline frontend/src/pages/Admin/Components/Resource/Resource.jsx:8]: Please add the three-dotted loader during the api call.

File: frontend/src/pages/Admin/Components/Resource/Resource.jsx [hunk 5]
```diff
@@ -1,9 +1,69 @@
...
+    const token = localStorage.getItem("token");
+    const fetchResource = async () => {
+      try {
+        const data = await axios.get(`${END_POINT}/resources/getresources`, {
+          headers: { Authorization: `Bearer ${token}` },
+        });
+        const resource = await data.data;
+        setResources(resource);
+      } catch (err) {
+        console.log("There was some error fetching resource page");
```
Comment (Kajol-Kumari) [inline frontend/src/pages/Admin/Components/Resource/Resource.jsx:19]: If any error occurs, can we please show a toast msg of errors instead?

**Comment by vinrusty** [comment]
Made the following changes:
-> Formatted the css files according to the specified convention.
-> Made the color of the cards same as the one in broadcast page.
-> Made the whole content of the card centered.
Screenshot (91)

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@vinrusty The page doesn't look responsive. Can we please fix that as well along with the below requested changes?

(Will update the level of the issue )

File: frontend/src/pages/Admin/Components/Resource/resource.module.scss (hunk 3 above)
Comment (vinrusty) [reply frontend/src/pages/Admin/Components/Resource/resource.module.scss]: removed the commented line

File: frontend/src/pages/Admin/Components/Resource/Resource.jsx (hunk 4 above)
Comment (vinrusty) [reply frontend/src/pages/Admin/Components/Resource/Resource.jsx:8]: ok on it

File: frontend/src/pages/Admin/Components/Resource/Resource.jsx (hunk 5 above)
Comment (vinrusty) [reply frontend/src/pages/Admin/Components/Resource/Resource.jsx:19]: ok on it

**Comment by vinrusty** [comment]
@Kajol-Kumari made the requested changes.
-> Added 3 dot loader.
-> Added Error toast

**Review by Kajol-Kumari — APPROVED** [review]
LGTM 👍

**Pull request merged**